### PR TITLE
Consistent output of shortened file paths

### PR DESCRIPTION
**Pull request by Roy-Orbison**
as per commit 2b7036ec684337a69d33d794b6f9652656bfde84

### CODE REVIEW

File: src/Phinx/Console/Command/AbstractCommand.php [hunk 1]
```diff
@@ -113,7 +113,7 @@ public function bootstrap(InputInterface $input, OutputInterface $output)
         $this->loadManager($input, $output);
 
         if ($bootstrap = $this->getConfig()->getBootstrapFile()) {
-            $output->writeln('<info>using bootstrap</info> .' . str_replace(getcwd(), '', realpath($bootstrap)) . ' ');
```
Comment (dereuromark) [inline src/Phinx/Console/Command/AbstractCommand.php:116]: I wonder if we should move the realpath() part also into that new method.

Comment (Roy-Orbison) [reply src/Phinx/Console/Command/AbstractCommand.php:116]: I'm not against it. If it were used everywhere `relativePath` is, I'd have included it.

Comment (dereuromark) [reply src/Phinx/Console/Command/AbstractCommand.php:116]: Sure, go ahead.

**Comment by dereuromark** [comment]
ping @Roy-Orbison

**Comment by Roy-Orbison** [comment]
I'll get to it.

**Comment by dereuromark** [comment]
Not a fan of Yoda ;) But I guess this is fair enough for now
Thanks a lot for the improvement!

**Comment by Roy-Orbison** [comment]
Not full Yoda, no `false === ...`.

More like spoken language to code: "Does the directory in question start with the current directory?"

**Pull request merged**